### DISCUSSION TITLE
Separate benchmarks into separate repo

### DISCUSSION THREAD

**Discussion by BenjaBobs** [thomhurst/TUnit#1431] [General]
The automated benchmarks seem to run once per day, which is quite cool, we have updated benchmark. :slightly_smiling_face: 
This however results in this repo never having a last commit time >24hr, and it makes it a bit more difficult to read through commit log for work being done.

If these automatic benchmarks were in a separate repo, it wouldn't add noise to the commit history of this one.
The benchmarks could also be stored, in case you want to draw graphs that show historic performances developments (I don't know if that's even on the map, but it would be cool).

Thoughts?

**Comment by vbreuss** [comment]
I think it would be sufficient to have it in a separate branch that is not merged into main. I had a similar idea in my project and have implemented the following workflow:
- Benchmarks are pushed to a `benchmarks` branch (see [here](https://github.com/aweXpect/aweXpect/blob/main/.github/workflows/build.yml#L82))
- When publishing Github Pages, I use the data from the `benchmarks` branch to [display a historic chart](https://awexpect.github.io/aweXpect/benchmarks) (see [here](https://github.com/aweXpect/aweXpect/blob/main/.github/workflows/build.yml#L248))
- Once in a while, I manually update the benchmarks branch

But in any case, this would mean, that the benchmarks would not be part of the README.md (or not up-to date).

**Reply by BenjaBobs** [reply]
If the charts / tables were rendered to .png files they could maybe be rendered in the main branch readme?
